mobile.html

```diff
@@ -145,24 +145,24 @@ <h1 class="title">About</h1>
 
     <section class="contact">
       <div class="headline">
-      <h2 class="title">Get Started</h2>
-      <p class="myname-intro"> I'm always interested in hearing about new projects, 
-                               so if you'd like to chat please get
-                               in touch.</p>
-      <form action="https://formspree.io/f/xgebneoy" method="post" class="contact-form">
+        <h2 class="title">Get started</h2>
+        <p class="myname-intro">I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</p>
+      </div>
+
+      <form action="https://formspree.io/f/xgebneoy" method="post">
         <div class="form-row">
-          <label for="name">Your name</label>
-          <input type="text" id="name" name="name" maxlength="30" placeholder="Write your name..." required>
+           <label for="name">Your name</label>
+           <input type="text" id="name" name="name" maxlength="30" placeholder="Write Your name..." required>
         </div>
         <div class="form-row">
           <label for="email">Your email</label>
-          <input type="email" id="email" name="email" placeholder="Write your email..." required>
-        </div>
-        <div class="form-row">
-          <label for="contact-descrip">Your message</label>
-          <textarea name="contact-descrip" id="contact-descrip" cols="30" rows="10" maxlength="500" placeholder="Enter your message here..." required></textarea>
-        </div>
-        <button type="button" class="button-box">Start collaboration</button>
+          <input type="email" id="email" name="email" placeholder="Write Your email..." required>
+       </div>
+       <div class="form-row">
+        <label for="contact-descrip">Your message</label>
+        <input type="textarea" id="contact-descrip" name="contact-descrip" cols="30" rows="10" placeholder="Enter your message here..." required>
+       </div>
+         <button class="button-box" type="button">Start collaboration</button>
```

Comment:
- [ ] Currently, your form is not getting submitted. This can be solved by adding an attribute `type="submit"` instead of `type="button"`  to your button element